src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+
+    //counting area of circle
+    @Override
+    public double countArea(Figure figure) {
+        return area = Math.PI * Math.pow(radius, 2);
```

Comment:
we dont need variable here
```suggestion
        return Math.PI * Math.pow(radius, 2);
```